Cargo.toml

```diff
@@ -16,11 +16,11 @@ anyhow = "1.0.40"
 winit = "0.24.0"
 skulpin = { version = "0.14.0", features = ["skia-complete", "winit-app", "winit-24"] }
 native-dialog = "0.5.5"
-image = "0.23.14"
+image = { version = "0.23.14", default-features = false, features = ["png", "webp"] }
```

Comment:
Please replace the `webp` feature with the `jpeg` feature. `image`'s WebP support is quite poor and we use the `webp` crate for WebP transcoding for transmission over the network.

We should include support for loading images of the most commonly used formats, and WebP is by no means common.

Alongside this removal, the list of supported extensions should also be updated inside the file picker when hosting a room from an image file.

Reply:
There you go.